src/AzSK.ADO/SVT/SVT.ps1

```diff
@@ -213,6 +213,11 @@ function Get-AzSKADOSecurityStatus
 		[Alias("ipt")]
 		$IterationPath,
 
+		[string]
+		[Parameter(Mandatory=$false)]
```

Comment:
Security severity of bugs to be logged.

Reply:
Specify the security severity of bugs to be logged.

Reply:
added